src/control/server/mgmt_pool_test.go

```diff
@@ -801,12 +801,20 @@ func TestServer_MgmtSvc_PoolDestroy(t *testing.T) {
 			}
 
 			if tc.expDrpcReq != nil {
```

Comment:
oh this is a bit of a mess, left over from trying to make the existing test cases in mgmt_pool_test.go work.